xonsh/xonfig.py

```diff
@@ -524,6 +524,7 @@ def _info(ns):
     )
     if ON_LINUX:
         data.append(("distro", linux_distro()))
+        data.append(("on wsl", bool(ON_WSL))),
```

Comment:
This will need to be imported from `xonsh.platform`

Reply:
Whoops, missed that, my bad.